.github/scripts/run-librispeech-zipformer-mmi-2022-12-08.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -e
+
+log() {
+  # This function is from espnet
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%d %H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+cd egs/librispeech/ASR
+
+repo_url=https://huggingface.co/Zengwei/icefall-asr-librispeech-zipformer-mmi-2022-12-08
+
+log "Downloading pre-trained model from $repo_url"
+git lfs install
```

Comment:
```suggestion
```

We are using 
```
GIT_LFS_SKIP_SMUDGE=1 git clone $repo_url
```
and
```
git lfs pull --include "data/*"
git lfs pull --include "exp/cpu_jit.pt"
git lfs pull --include "exp/pretrained.pt"
```
so it is fine to remove `git lfs install`.

Reply:
OK. Thanks.